apps/site/src/views/Home/components/StickerLayout/StickerLayout.tsx

```diff
@@ -0,0 +1,24 @@
+import { HackSticker, HeartSticker } from "@/components/Sticker/Stickers";
+
+export default function StickerLayout() {
+	return (
+		<div style={{ maxWidth: "100vw" }}>
+			<HackSticker
+				style={{
+					position: "absolute",
+					left: "60vw",
+					top: "15vh",
+					zIndex: 100,
+				}}
+			/>
```

Comment:
Issue: The Hack sticker is already being used in the index card section. Since we'll be putting more stickers here on the top soon (according to the design), it might be better to keep just the heart sticker and have it positioned at the bottom right of the button.